README.md

```diff
@@ -160,6 +160,9 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the keyboard. If it's something you think is unfixable then use
   `$FLOW_IGNORE` instead. Please explain your rationale in the comment and link
   to a GitHub issue if possible.
+* If you're stumped on why some Flow annotations aren't working, try using
```

Comment:
Keep in mind that `yarn flow --stuff` doesn't work and that's where things get annoying. You have to escape out of yarn with dashes: `yarn flow -- --stuff --other-options`. It works fine for sub-commands though.